ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -255,3 +255,40 @@ SENSOR_ZTM_ZHM_CO2M_VOCM_PMM:
   - CO2M
   - VOCM
   - PMM
+
+SENSOR_ZHM_ZOC:
+  description: "Zone occupant counter with zone humidity monitoring."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZHM
+  - ZOC
+
+SENSOR_ZTM_ZHM_CO2M_VOCM_PML_SPLM:
+  description: "Indoor Air Quality sensor with temperature, humidity, CO2, VOC, particulates (density), illuminance and sound monitoring."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZTM
+  - ZHM
+  - CO2M
+  - VOCM
+  - SPLM
+  uses:
+  - zone_air_aqi_sensor
+  - illuminance_sensor
+  - zone_air_pm2pt5_density_sensor
+  - zone_air_pm10pt0_density_sensor
+
+SENSOR_ZTM_ZHM_CO2M_VOCM_PML:
```

Comment:
This is a different type of sensor, but measures very similar values to `SENSOR_ZTM_ZHM_CO2M_VOCM_PMM`, except it measures `density` of particulates, rather than `concentration`. Should an exisitng entity type be used/extended to implement the density fields, and mark the concenatration ones as missing?